src/nft/components/bag/Bag.tsx

```diff
@@ -320,17 +352,17 @@ const Bag = () => {
               Continue
             </Box>
           )}
-        </Column>
+        </BagContainer>
       ) : (
-        <Column zIndex={isMobile || isOpen ? 'modalOverTooltip' : '3'} className={styles.bagContainer}>
+        <BagContainer raiseZIndex={isMobile || isModalOpen}>
           <ListingModal />
-        </Column>
+        </BagContainer>
       )}
 
       {isDetailsPage ? (
         <DetailsPageBackground onClick={toggleBag} />
       ) : (
-        isOpen && <Overlay onClick={() => (!bagIsLocked ? setModalIsOpen(false) : undefined)} />
+        isModalOpen && <Overlay onClick={() => (!bagIsLocked ? setModalIsOpen(false) : undefined)} />
```

Comment:
The overlay appeared for me when I started selecting items from my bag which blocks me from selecting more (until I click on the overlay)
<img width="1725" alt="Screen Shot 2022-12-13 at 08 56 05 " src="https://user-images.githubusercontent.com/11512321/207390258-4d024660-cfe5-41b3-be26-c97a47b4e24a.png">

Reply:
this is actually a bug on main - I am going to resolve in a sep pr. I currently don't know how to repro though it happens randomly for me